docs/elements/hx-search-control/index.html

```diff
@@ -0,0 +1,144 @@
+---
+title: <hx-search-control>
+minver: 0.18.0
+controlElement: <input type="search">
+also:
+  components/search: Search
+  elements/hx-search: <hx-search>
+  elements/hx-search-assistance: <hx-search-assistance>
+---
+{% extends 'element.njk' %}
+{% import '_utils.njk' as utils %}
+{% block page_header %}
+  <p>
+    The <code>{{page.title}}</code> block element is a container that augments
+    native browser validation and styles for an
+    <code>&lt;input type="search"&gt;</code> element.  Used to build
```

Comment:
Last sentence is a fragment. Please update to `It is used to build...`

Reply:
Sentence updated.